README.md

```diff
@@ -126,18 +135,20 @@ Standard IntelliJ dialog for interactive rebase will appear.
 </details>
 
 <details>
-<summary>Push</summary>
+<summary><strong>Push</strong></summary>
+<br/>
 
 After a rebase, you might want to push the rebased branch to the remote.
 Right-click on the given branch and select `Push (Current) Branch` from a context menu.
 Push dialog will appear with a proper push button (regular push or force push) depending on which of them is needed.
-Basically, we need force push in a case when we rebased our branch because local and remote branch diverged from each other (they have different commits), so git won't let us do regular push.
+Basically, you need force push in a case when you rebased our branch because local and remote branch diverged from each other (they have different commits), so git won't let us do a regular push.
```

Comment:
```suggestion
Basically, you need force push in a case when you rebased your branch because local and remote branch diverged from each other (they have different commits), so git won't let you do a regular push.
```